tests/functional/test_httplib2.py

```diff
@@ -64,10 +66,11 @@ def test_httpretty_provides_easy_access_to_querystrings(now):
 
 
 @httprettified
-@within(two=microseconds)
-def test_httpretty_should_mock_headers_httplib2(now):
+@patch('httpretty.core.datetime')
```

Comment:
this would be better with [freezegun](https://pypi.org/project/freezegun/)

Reply:
Done.